### PR TITLE
A0-1784 Redirect Devloper/Contracts to an external webpage

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
Most of the PR is removal of `page-contracts`. 
Change works since in https://github.com/Cardinal-Cryptography/apps/blob/alephzero/packages/apps/src/Menu/Item.tsx#L28 there's already functionality which enables external link instead of an internal route, so we just need to add an optional `href` property to `Route` 
Also I disabled the account to check in the wallet for Contracts since it does not make sense to enforce it now.

### CODE REVIEW

File: packages/apps-routing/src/contracts.ts [hunk 1]
```diff
@@ -23,13 +23,14 @@ export default function create (t: TFunction): Route {
   return {
     Component,
     display: {
-      needsAccounts: true,
+      needsAccounts: false,
```
Comment (deuszx) [inline packages/apps-routing/src/contracts.ts:34]: OMG, it was that easy? 🤯 

There is tone of value in being able to call read-only methods w/o having to connecting an account.

File: packages/apps-routing/src/contracts.ts [hunk 2]
```diff
@@ -23,13 +23,14 @@ export default function create (t: TFunction): Route {
...
       needsApi: [
         'tx.contracts.instantiateWithCode'
       ],
       needsApiCheck
     },
     group: 'developer',
+    href: 'http://contracts-ui.substrate.io/',
```
Comment (deuszx) [inline packages/apps-routing/src/contracts.ts]: Why not `https`?

Comment (h4nsu) [reply packages/apps-routing/src/contracts.ts]: Contracts UI is a generic page with multiple possible backends, kinda like polkadot.js.org. Would be nice to set target backend directly, something along the lines:
`https://contracts-ui.substrate.io/?rpc=wss://ws.test.azero.dev`
Of course, if possible, we would like to have:
`https://contracts-ui.substrate.io/?rpc=[WEBSOCKET WE ARE NOW CONNECTED TO]`

**Review by h4nsu — CHANGES_REQUESTED** [review]

File: packages/apps-routing/src/contracts.ts (hunk 2 above)
Comment (deuszx) [reply packages/apps-routing/src/contracts.ts]: Oh yeah, the second idea is very good :+1:

Comment (Marcin-Radecki) [reply packages/apps-routing/src/contracts.ts]: Done

File: packages/apps-routing/src/contracts.ts (hunk 1 above)
Comment (Marcin-Radecki) [reply packages/apps-routing/src/contracts.ts:34]: Yes, easy

**Review by h4nsu — APPROVED** [review]
Awesome, thanks!

**Review by deuszx — APPROVED** [review]

**Pull request merged**